modules/vpn/example/main.tf

```diff
@@ -0,0 +1,106 @@
+data "google_compute_network" "test" {
+  name    = "j26-vpc-app01-data-nonprod"
+  project = "gcp-gcs-pso"
+}
+
+module "vpn" {
+  source = "../"
+
+  project = "gcp-gcs-pso"
+  region  = "us-central1"
+
+  vpn_gateway_name = "my-test-gateway"
+  router_name      = "my-test-router"
+  network          = data.google_compute_network.test.self_link
+
+  vpn_config = {
```

Comment:
Maybe we can move some of this config into tfvars ?

Reply:
Sure. That's an obvious oversight on my side. Fixed